packages/router/test/integration/SwaggerRouter.integration.ts

```diff
@@ -3,13 +3,13 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {SwaggerRouter} from '../../../lib/router/SwaggerRouter';
+import {SwaggerRouter} from '../..';
 import * as http from 'http';
 import * as request from 'request-promise';
-import { FullRequestResponse } from './../../support/FullRequestResponse';
+import { FullRequestResponse } from '../../../loopback/test/support/FullRequestResponse'; // FIXME(bajtos)
```

Comment:
Fixing this (and L12 below) properly requires a bit more of work, let's leave it up for later (as part of #135).